### ISSUE TITLE
User defined order of notes

### ISSUE THREAD

**Issue by MarkusSteinbrecher** [bldrs-ai/Share#311]
At the moment, the order in which notes are shown is defined by the order in which they are created in Github. That means that updates on notes are no fun ... As a user I want to be able to define the order of the notes - possibly through some metadata in the note itself (e.g. #nr-1)

**Comment by pablo-mayrgundter** [comment]
How about the sorting available from github? 
<img width="351" alt="image" src="https://user-images.githubusercontent.com/2480879/182663012-c8041511-c53a-4409-a2f1-bffa24cc2a31.png">

**Comment by MarkusSteinbrecher** [comment]
What if I create 10 issues and then decide at the end that I need another one in the middle? I don't think the standard sorting is enough for this.

**Comment by pablo-mayrgundter** [comment]
Ah, that sounds like the portfolio use-case where you're really trying to present information in a certain order.  Maybe we can integrate these with a single solution

**Comment by OlegMoshkovich** [comment]
@pablo - do you oppose implementing a meta string in the note itself?
'possibly through some metadata in the note itself (e.g. #nr-1)'

**Comment by pablo-mayrgundter** [comment]
Yeah, that's reasonable.  Just want to make sure we keep the use-cases differentiated.

Like, let's say we do this so that we can achieve a presentation function.  That's cool for presentations, but then someone might come along and look at a set of notes and expect them to be sorted by post-time, i.e. as a comment thread.  At this point, the explicit ordering becomes complicated to think about.  How does the user control it?  How do they know if it's active?

So I propose prioritizing the development the modal switch.  A user should be able to use comments to build a presentation, but there's a clear modal shift to switch into presentation and out of it.  And for this feature, when not in presentation mode, the comment ordering tags would be ignored.

**Comment by OlegMoshkovich** [comment]
Feature Request.
Putting it into Notes backlog

**Comment by ronplaymeta** [comment]
It seems like good to use GitHub itself sorting feature too with presentation mode by using modal switch.
